R/parameters.R

```diff
@@ -180,9 +180,8 @@ generate_parameters <- function(
                          D0 = R0,
                          trans_increase = trans_increase_array,
                          trans_profile = trans_profile_array,
-                         beta_list = beta,
-                         beta = beta[1], # When odin code is updated, set this list
-                         beta_dates = beta_dates,
+                         beta_y = beta,
```

Comment:
Is it worth adding in a similar set to `NULL` in `test-parameters.R` to ignore the betas for the time being, and still get the tests passing. Looks like all the others are fine

Reply:
Yup, added that now

Reply:
Shouldn't it be D0=D0 rather R0?

Reply:
Nice spot Marc, though of course they're the same here at the moment so this had no effect.  Fixed now